src/main/java/uk/gov/register/resources/v2/EntryResource.java

```diff
@@ -34,7 +37,7 @@ public EntryResource(RegisterReadOnly register, ViewFactory viewFactory, Request
         this.register = register;
         this.viewFactory = viewFactory;
         this.requestContext = requestContext;
-        this.httpServletResponseAdapter = new HttpServletResponseAdapter(requestContext.httpServletResponse);
+        this.httpServletResponseAdapter = new HttpServletResponseAdapter(requestContext.getHttpServletResponse());
```

Comment:
what's the purpose of this change? Was it intentional?

Reply:
It's because ReqeustContext is now in a different package to the v1 and v2 packages, so the field can't be accessed directly anymore.